packages/sui-tealium-tracker/src/FunctionThrottler.js

```diff
@@ -0,0 +1,15 @@
+export class FunctionThrottler {
```

Comment:
Is this behavior different than https://www.npmjs.com/package/just-throttle ?

Reply:
Don't really know. What this function does is to delay a function call if was previously called before XXX seconds. First execution will always be done instant.

just-throttle does the same?